packages/web/src/app/[domain]/components/pathHeader.tsx

```
@@ -204,19 +208,24 @@ export const PathHeader = ({
+                        <a href={info.repoLink} target="_blank" rel="noopener noreferrer">
+                            <Image
+                                src={info.icon}
+                                alt={info.codeHostName}
+                                className={`w-4 h-4 ${info.iconClassName}`}
+                            />
+                        </a>
+                    ) : (
+                        <LaptopIcon className="w-4 h-4" />
+                    )}
+                </>
             )}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)